src/scc_hypervisor_collector/api/config_manager.py

```diff
@@ -124,6 +124,16 @@ def _remove_idless_duplicates(backend: Dict[str, Any],
                       'backend with id %s', repr(existing_id))
             backends.remove(no_id_backend)
 
+    @staticmethod
+    def _get_backends(cfg: Dict[str, Any]) -> List[Dict[str, Any]]:
+        """ Return the list of backends - an empty list if
+        backends section is empty
+        """
+        temp: List[Dict[str, Any]] = []
```

Comment:
Since the variable is going to be holding the backends we should call it `backends` rather than `temp`; when I said temp variable in our discussions I wasn't actually meaning to call it temp, but more that it was a variable that would be temporary in nature ;-)

I would change this pattern slightly; in the current form it will call get() twice, and while a good code optimiser will avoid the unnecessary overhead, with a little tweak we can handle it by calling get() only once, which reads better.

    backends: List[Dict[str, Any]] = cfg.get('backends', [])
    if backends is None:
        backends = []
    return backends.copy()

The other advantage of this pattern are that we only assign to backends once, in the normal/expected case of actually getting a correctly formatted input; it is usually better to handle the expected case and the normal code path, and deal with exceptional circumstances as the alternate code path.